tests/playwright/specs/plugins/01-Plugins.spec.ts

```diff
@@ -0,0 +1,588 @@
+import { expect, type Locator } from '@playwright/test'
+import baseTest from '@pw/base-test'
+import { autocompleteDeleteModal } from '@pw/commands/autocompleteDeleteModal'
+import { clearKongResources } from '@pw/commands/clearKongResources'
+import { clickEntityListAction } from '@pw/commands/clickEntityListAction'
+import { createKongResource } from '@pw/commands/createKongResource'
+import { fillEntityForm } from '@pw/commands/fillEntityForm'
+import { switchDetailTab } from '@pw/commands/switchDetailTab'
+import { waitAndDismissToasts } from '@pw/commands/waitAndDismissToast'
+import { withNavigation } from '@pw/commands/withNavigation'
+import { getPropertyValue } from '@pw/commands/getPropertyValue'
+import { ConsumerListPage } from '@pw/pages/consumers'
+import { PluginListPage } from '@pw/pages/plugins'
+import { RouteListPage } from '@pw/pages/routes'
+import { ServiceListPage } from '@pw/pages/services'
+
+const mockConsumerName = 'testUser'
+const mockRouteName = 'testRoute'
+const mockServiceName = 'testService'
+const mockTag = 'testTag'
+const mockPluginName = 'prometheus'
+const mockInstanceName = 'myCustomName'
+
+const test = baseTest().extend<{
+  consumerListPage: ConsumerListPage,
+  pluginListPage: PluginListPage
+  serviceListPage: ServiceListPage
+}>({
+  consumerListPage: async ({ page }, use) => use(new ConsumerListPage(page)),
+  pluginListPage: async ({ page }, use) => use(new PluginListPage(page)),
+  serviceListPage: async ({ page }, use) => use(new ServiceListPage(page)),
+})
+
+test.describe('plugins', () => {
+  test.beforeAll(async () => {
+    await clearKongResources('/routes')
+    await clearKongResources('/services')
+    await clearKongResources('/consumers')
+    await clearKongResources('/plugins')
+  })
+
+  test.beforeEach(async ({ page }) => {
+    await new PluginListPage(page).goto()
+  })
+
+  test.afterAll(async () => {
+    await clearKongResources('/routes')
+    await clearKongResources('/services')
+    await clearKongResources('/consumers')
+    await clearKongResources('/plugins')
+  })
+
+  test('plugin list should be empty now', async ({ page, pluginListPage }) => {
+    await pluginListPage.goto()
```

Comment:
We already have `await new PluginListPage(page).goto()` in `beforeEach`, thus if `await pluginListPage.goto()` appears in the first line of a test, it could be safely removed.

Reply:
Removed in https://github.com/Kong/kong-manager/pull/49/commits/992affd805b4ae47571506fb224afc284f072134